ebuild-writing/misc-files/patches/text.xml

```diff
@@ -112,6 +112,42 @@ from the <c>vim</c> patch tarball:
 </body>
 </section>
 
+<section>
+<title>Conditional Patching</title>
+<body>
+<p>
```

Comment:
Please add a blank line before `<p>` and `<codesample>` tags, see our [style guide](https://devmanual.gentoo.org/appendices/devbook-guide/index.html#internal-coding-style).

Reply:
Done.